organic-search-analytics/report.php

```diff
@@ -88,7 +88,7 @@
 		<?php } ?>
 
 		<?php
-			$reportQuery = "SELECT " . $groupBy . ", count(" . ( $groupBy != "query" ? 'DISTINCT ' : '' ) . "query) as 'queries', sum(impressions) as 'impressions', sum(clicks) as 'clicks', avg(avg_position) as 'avg_position' FROM ".$mysql::DB_TABLE_SEARCH_ANALYTICS." " . $reportDetails['whereClauseTable'] . "GROUP BY " . $groupBy . " ORDER BY " . $reportDetails['sortBy'] . " ASC";
+			$reportQuery = "SELECT " . $groupBy . ", count(" . ( $groupBy != "query" ? 'DISTINCT ' : '' ) . "query) as 'queries', sum(impressions) as 'impressions', sum(clicks) as 'clicks', sum(avg_position*impressions)/sum(impressions) as 'avg_position' FROM ".$mysql::DB_TABLE_SEARCH_ANALYTICS." " . $reportDetails['whereClauseTable'] . "GROUP BY " . $groupBy . " ORDER BY " . $reportDetails['sortBy'] . " ASC";
```

Comment:
I'm not sure that I agree that this is a safe modification.

- Can you please explain this change in further detail?
- What is the intended purpose of this change?
- Can you provide an example illustrating the issue and how this solves the issue?

My thoughts on why this is not a necessary change:

- The data retrieved from Google is already an average based on the number of impressions
- This is only the select of data - calculations are not done at this time (though maybe could be...)
- Further down in the code calculations on the average are made - wouldn't this result in calcuating the average twice?
  - https://github.com/qstarx/organic-search-analytics/blob/ef5a9b36e4e33f4b36c12ab1b718097a34028587/organic-search-analytics/report.php#L114-L116

Thanks!